lib/routes/routeBackbeat.js

```diff
@@ -339,7 +364,7 @@ function putObject(request, response, log, callback) {
                 return callback(errors.BadDigest);
             }
             const dataRetrievalInfo = {
-                versionId: retrievalInfo.dataStoreVersionId,
+                versionId: retrievalInfo.dataStoreVersionId || '',
```

Comment:
Can you add a todo here, mentioning to remove when Azure versioning is implemented.

Reply:
Updated.